package.json

```diff
@@ -29,29 +29,29 @@
   },
   "homepage": "https://github.com/digicatapult/openapi-service-template#readme",
   "dependencies": {
-    "body-parser": "^1.19.0",
+    "body-parser": "^1.20.0",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "dotenv": "^8.2.0",
-    "envalid": "^7.0.0",
-    "express": "^4.17.1",
-    "express-openapi": "^10.0.0",
-    "pino": "^6.11.1",
-    "pino-http": "^5.5.0",
-    "swagger-ui-express": "^4.2.0"
+    "dotenv": "^16.0.0",
```

Comment:
Its probably worth noting this here
https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md

However as this is a fresh repository there is no expectation that there are .env files currently existing.